diff --git a/Makefile.in b/Makefile.in
--- a/Makefile.in
+++ b/Makefile.in
@@ -515,27 +515,10 @@
 	    fi; \
 	  done; \
 	done; rm -f c$${pid}_.???; exit $$bad
-install-logDATA: $(log_DATA)
-	@$(NORMAL_INSTALL)
-	@list='$(log_DATA)'; test -n "$(logdir)" || list=; \
-	if test -n "$$list"; then \
-	  echo " $(MKDIR_P) '$(DESTDIR)$(logdir)'"; \
-	  $(MKDIR_P) "$(DESTDIR)$(logdir)" || exit 1; \
-	fi; \
-	for p in $$list; do \
-	  if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
-	  echo "$$d$$p"; \
-	done | $(am__base_list) | \
-	while read files; do \
-	  echo " $(INSTALL_DATA) $$files '$(DESTDIR)$(logdir)'"; \
-	  $(INSTALL_DATA) $$files "$(DESTDIR)$(logdir)" || exit $$?; \
-	done
+install-logDATA:
 
-uninstall-logDATA:
-	@$(NORMAL_UNINSTALL)
-	@list='$(log_DATA)'; test -n "$(logdir)" || list=; \
-	files=`for p in $$list; do echo $$p; done | sed -e 's|^.*/||'`; \
-	dir='$(DESTDIR)$(logdir)'; $(am__uninstall_files_from_dir)
+uninstall-logDATA:
+
 install-logrotateDATA: $(logrotate_DATA)
 	@$(NORMAL_INSTALL)
 	@list='$(logrotate_DATA)'; test -n "$(logrotatedir)" || list=; \
@@ -557,27 +540,10 @@ uninstall-logrotateDATA:
 	@list='$(logrotate_DATA)'; test -n "$(logrotatedir)" || list=; \
 	files=`for p in $$list; do echo $$p; done | sed -e 's|^.*/||'`; \
 	dir='$(DESTDIR)$(logrotatedir)'; $(am__uninstall_files_from_dir)
-install-pkglocalstateDATA: $(pkglocalstate_DATA)
-	@$(NORMAL_INSTALL)
-	@list='$(pkglocalstate_DATA)'; test -n "$(pkglocalstatedir)" || list=; \
-	if test -n "$$list"; then \
-	  echo " $(MKDIR_P) '$(DESTDIR)$(pkglocalstatedir)'"; \
-	  $(MKDIR_P) "$(DESTDIR)$(pkglocalstatedir)" || exit 1; \
-	fi; \
-	for p in $$list; do \
-	  if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
-	  echo "$$d$$p"; \
-	done | $(am__base_list) | \
-	while read files; do \
-	  echo " $(INSTALL_DATA) $$files '$(DESTDIR)$(pkglocalstatedir)'"; \
-	  $(INSTALL_DATA) $$files "$(DESTDIR)$(pkglocalstatedir)" || exit $$?; \
-	done
+install-pkglocalstateDATA:
 
-uninstall-pkglocalstateDATA:
-	@$(NORMAL_UNINSTALL)
-	@list='$(pkglocalstate_DATA)'; test -n "$(pkglocalstatedir)" || list=; \
-	files=`for p in $$list; do echo $$p; done | sed -e 's|^.*/||'`; \
-	dir='$(DESTDIR)$(pkglocalstatedir)'; $(am__uninstall_files_from_dir)
+uninstall-pkglocalstateDATA:
+
 install-sysconfigDATA: $(sysconfig_DATA)
 	@$(NORMAL_INSTALL)
 	@list='$(sysconfig_DATA)'; test -n "$(sysconfigdir)" || list=; \
